index.js

```diff
@@ -37,6 +37,7 @@ const camundaCloud13Rules = withConfig(camundaCloud12Rules, { version: '1.3' });
 
 const camundaCloud80Rules = withConfig({
   ...omit(camundaCloud13Rules, 'no-template'),
+  'version-compatibility-validator': 'warn'
```

Comment:
We probably want to find a better name for this.

Reply:
Waiting for @philippfromme to share his cents.

Reply:
This definitely needs to be renamed. The scope of the plugin is bigger than just connector templates so we should namespace connector template specific things.